HPXMLtoOpenStudio/resources/hpxml.rb

```diff
@@ -90,12 +90,17 @@ class HPXML < Object
   FoundationTypeSlab = 'SlabOnGrade'
   FrameFloorOtherSpaceAbove = 'above'
   FrameFloorOtherSpaceBelow = 'below'
+  FuelLoadTypeGrill = 'grill'
+  FuelLoadTypeLighting = 'lighting'
+  FuelLoadTypeFireplace = 'fireplace'
   FuelTypeElectricity = 'electricity'
   FuelTypeNaturalGas = 'natural gas'
   FuelTypeOil = 'fuel oil'
   FuelTypePropane = 'propane'
   FuelTypeWood = 'wood'
   FuelTypeWoodPellets = 'wood pellets'
+  HotTubHeaterTypeElectric = 'electric resistance'
+  HotTubHeaterTypeGas = 'gas fired'
```

Comment:
Applies to pool heaters too, right? Call them HeaterTypeElectric and HeaterTypeGas.